charts/backstage/Chart.yaml

```diff
@@ -41,4 +45,4 @@ sources:
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.15.2
+version: 2.16.0
```

Comment:
I think the version # needs to be bumped in more than one place...

Reply:
released in https://artifacthub.io/packages/helm/rhdh-chart/backstage

fixes in followup PR:
https://github.com/redhat-developer/rhdh-chart/pull/16